### PR TITLE
Create selector to filter out deactiviated realm emojis.

### PR DESCRIPTION
**Pull request by jainkuniya**

### CODE REVIEW

File: src/emoji/emojiSelectors.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* @flow */
+import { createSelector } from 'reselect';
+import { getRealmEmoji } from '../selectors';
+
+export const getActiveRealmEmoji = createSelector(getRealmEmoji, emojis =>
+  Object.keys(emojis).reduce((list, key) => {
```
Comment (borisyankov) [inline src/emoji/emojiSelectors.js]: Use `filter`

**Comment by borisyankov** [comment]
Address my comment and fix the tests, before I merge this.

**Comment by borisyankov** [comment]
Actually, what is the point of filtering the emojis?

**Comment by jainkuniya** [comment]
> Actually, what is the point of filtering the emojis?

To prevent users to use outdated/deactivated emojis.
(basically if we show outdated emojis in autocomplete, then users can select them and expect that emoji_name (`:deactivated_emoji`) will convert to emoji after sending message (which will not).  Just to make sure it is bug free :))

**Comment by borisyankov** [comment]
Cool, thanks for clarifying.

**Pull request merged**